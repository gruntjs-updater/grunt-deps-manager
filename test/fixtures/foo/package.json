{
    "name": "foo",
    "dependencies": {
        "should-change": "1.2.3"
    },
    "devDependencies": {
        "should-change-too": "4.5.6"
    }
}