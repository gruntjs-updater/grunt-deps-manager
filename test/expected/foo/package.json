{
  "name": "foo",
  "dependencies": {
    "should-change": "0.0.8"
  },
  "devDependencies": {
    "should-change-too": "0.0.8"
  }
}